build-support/bin/native/bootstrap.sh

```diff
@@ -114,6 +114,13 @@ function bootstrap_native_code() {
     source "${cffi_env_script}"
     local readonly native_binary="$(build_native_code)"
 
+    # If bootstrapping the native engine fails, don't attempt to run pants
+    # afterwards.
+    if [ -z $native_binary ]
```

Comment:
Currently it returns either a file name or an empty string. I thought that checking for an empty string would make more sense since the result isn't always a file.

That said I tried `[[ -f "" ]]` and it does eval to false, so I could use that instead. Using `-f` would cover the case where running the build passed, but for some reason the file wasn't created and conveys the fact that native_binary should be a file name.

I guess that's all a long way of saying, "Yes it should."